### PR TITLE
=)

### PR DESCRIPTION
**Pull request by fredrikbryntesson**

### CODE REVIEW

File: =) [hunk 1]
```diff
@@ -0,0 +1,8 @@
+=)
+=)
+=)
+=)
+=)
```
Comment (marcusnaslund) [inline =)]: This line causes segfaults.

**Comment by sebastianbaginski** [comment]
This should go upstream. We need more happiness asap.

**Comment by marcusnaslund** [comment]
Will you peer review @sebastianbaginski ? All tests check out and magic has no complaints.

**Comment by marcusnaslund** [comment]
Passes validation. The phone seems a lot happier.

**Pull request closed without merging**